_toc.yml

```diff
@@ -52,7 +52,12 @@ entries:
       - file: docs/tools/terraform/index
         title: Aiven Terraform provider
         entries:
-        - glob: docs/tools/terraform/*
+        - file: docs/tools/terraform/upgrade-provider-v1-v2
+        - file: docs/tools/terraform/reference/index
+          entries:
+          - file: docs/tools/terraform/reference/cookbook/index
```

Comment:
I don't think I wrote clear instructions! This should be `cookbook.rst` not `cookbook/index.rst`, just for consistency with how the concept/howto/reference sections are done in the product sections.

The same thing applies to the `reference/index.rst` - it should be `reference.rst`.